pkg/exposer/csi_snapshot.go

```diff
@@ -204,6 +204,7 @@ func (e *csiSnapshotExposer) GetExposed(ctx context.Context, ownerObject corev1.
 
 	backupPodName := ownerObject.Name
 	backupPVCName := ownerObject.Name
+	volumeName := string(ownerObject.UID)
```

Comment:
Why the `volumeName` is equal to `ownerObject.UID`?

Reply:
Because we name the volume as `ownerObject.UID` when creating the backup pod, so it is always true that the backup volume is with the name of `ownerObject.UID` in the backup pod.
See the `createBackupPod` function for how the backup pod is created.